ReportingServicesTools/Functions/CatalogItems/Update-RsSubscription.ps1

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+
+function Update-RsSubscription

```

Comment:
Could you please rename this to something like Set-RsSubscription? It would be more consistent with the other update PowerShell cmdlets we have.

Reply:
Set-RsSubscription already exists in the module